client/src/components/graph/setupSVGandBrush.js

```diff
@@ -21,10 +21,12 @@ export default (
   const svg = d3
     .select("#graphAttachPoint")
     .append("svg")
+    .attr("id", "tool")
```

Comment:
@colinmegill going to defer to you on this, but tool seems like a pretty generic id, maybe graph-tools?